systemd/systemd.go

```diff
@@ -932,6 +932,7 @@ var squashfsFsType = squashfs.FsType
 var mountUnitTemplate = `[Unit]
 Description=Mount unit for %s, revision %s
 Before=snapd.service
+After=zfs-mount.service
```

Comment:
Let's leave a comment in the code somewhere that we workaround LP:xxx this way and that we believe it's a zfs-linux/systemd bug but we do it out of courtesy  because it's "cheap" for us.